src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -65,7 +65,17 @@ const StyledTextBlock = styled(TextBlock)`
   align-self: center;
 `;
 
-const Alert = ({type, icon, iconSize, children, className, ...props}) => {
+type AlertProps = {
+  type?: string;
+  icon?: string;
+  iconSize?: string;
+  children: React.ReactNode;
+  className?: string;
+  alignTop?: boolean;
+  system?: boolean;
+};
```

Comment:
It needs div element types too https://github.com/getsentry/sentry/pull/14703/files#diff-3d15e1f6a6441ac3fdf4f164638bcaefR19